lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -2353,7 +2345,8 @@ galaxy:
   # is relative to galaxy/static
   #welcome_directory: plugins/welcome_page/new_user/static/topics/
 
-  # Vault config file
+  # Vault config file.
   # The value of this option will be resolved with respect to
   # <config_dir>.
-  vault_config_file: vault_conf.yml
+  #vault_config_file: vault_conf.yml
```

Comment:
@nsoranzo that's the issue, we now don't have
```yaml
galaxy:
  vault_config_file: vault_conf.yml
```
but
```yaml
galaxy:
```
which is None, and gravity checks for None in https://github.com/galaxyproject/gravity/blob/main/gravity/config_manager.py#L352

We can probably just make sure there's a galaxy key and not make any assumptions about the value.

Reply:
Exactly, came to the same conclusion! I can open a PR for Gravity, if you haven't one ready yet.

Reply:
Nope, thank you!